.github/workflows/build.yml

```diff
@@ -13,19 +13,28 @@ on:
 jobs:
   build:
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
+      max-parallel: 1
```

Comment:
It feels like this should not be required, do you know why it is?

Reply:
It appeared to lessen the flapping tests by working in serial, however I concede this could have been co-incidental. It does have an impact on the processing time, but is this a time dependant task?

Reply:
It appeared to lessen the flapping tests by working in serial, however I concede this could have been co-incidental. It does have an impact on the processing time, but is this a time dependant task?

Reply:
No they are not really time sensitive (until Microsoft starts charging for cycles) but changes like these will stay forever until a problem pops up because we will forget why its there and assume it is required. I'm fine leaving it in, just noting that I will promptly forget this entire discussion and treat it like gospel 🤣

Reply:
In that case, maybe it would be better to remove the serialisation and work on fixing the flapping tests instead.